### ISSUE TITLE
Encoding issue (utf-8) of German umlauts on Windows

### ISSUE THREAD

**Issue by dsvanidze** [posit-dev/py-shiny#1231]
I have encoding issues when I use German special letters in the Shiny app on Windows. It renders them incorrectly in a browser and also in the terminal output.

To reproduce, app/app.py file has just two lines of code:
```python
from shiny.express import input, ui, render
print("üäöß")
```
When I run this code with Python `python app/app.py`, it prints the letters correctly. However, when I run the Shiny App with `shiny run --reload --launch-browser app/app.py`, it prints the following `Ã¼Ã¤Ã¶ÃŸ`. 

I just use print() here, but exactly the same happens with the string, when I use Shiny components to show text. This issue does not occur, when I run the app on WSL. 

How can I fix the problem on Windows so that German text can be rendered correctly?

**Comment by wch** [comment]
We recently merged a fix (#1203) for encoding issues on Windows.

Can you try installing the development version from github and see if it helps?

```
pip install https://github.com/posit-dev/py-shiny/tarball/main
```

**Comment by dsvanidze** [comment]
@wch thank you for your reply! I did it and this solved the issue 🥳
